tests/patterns/kotlin/class_newline.kt

```diff
@@ -0,0 +1,2 @@
+// ERROR:
+class MyClass constructor(arg: Int) {}
```

Comment:
Could you add a test case like the parsing test above to this file as well? The parsing test ensures that we get a valid parse, but it doesn't ensure that we actually get a sensible AST.

Reply:
Done.